### PR TITLE
Refactor for ballerina lang updates and Update toml file

### PR DESCRIPTION
**Pull request by ldclakmal**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,84 +1,44 @@
...
-| Language Version           | Endpoint Version    | Twilio Basic API Version | Twilio Authy API Version |
-| -------------------------- | ------------------- | ------------------------ | ------------------------ |
-| 0.970.0-beta1-SNAPSHOT     | 0.5.10              | 2010-04-01               | v1                       |
-
-##### Prerequisites
-
-1. Download the ballerina [distribution](https://ballerinalang.org/downloads/).
+| Ballerina Language Version  | Twilio API Version | Twilio Basic API Version | Twilio Authy API Version |
+| :--------------------------:|:------------------:|:------------------------:|:------------------------:|
+| 0.970.0-beta2               | 0.8.0              | 2010-04-01               | v1                       |
```
Comment (muthulee) [inline README.md]: Please remove the first column.

File: README.md [hunk 2]
```diff
@@ -1,84 +1,44 @@
...
 
-2. Create a Twilio account (https://www.twilio.com/) and obtain the following parameters:
-* ACCOUNT SID
-* AUTH TOKEN
+## Getting started
 
-3. Create a Twilio Authy app withing Twilio account (https://www.twilio.com/console/authy/applications) and obtain the
-following parameters:
-* PRODUCTION API KEY
+1.  Refer https://ballerina.io/learn/getting-started/ to download and install Ballerina.
```
Comment (muthulee) [inline README.md]: Put link words. Refer Getting Started guide to download and install Balelrina.

File: README.md [hunk 3]
```diff
@@ -1,84 +1,44 @@
...
-##### Example
-* Request 
-```
+```ballerina
     import wso2/twilio;
 
-    public function main (string[] args) {
+    function main (string... args) {
         endpoint twilio:Client twilioClient {
              accountSid:config:getAsString(ACCOUNT_SID),
```
Comment (muthulee) [inline README.md]: Give an indication that ACCOUNT_SID needs to be changed.

File: README.md (hunk 1 above)
Comment (ldclakmal) [reply README.md]: Since this is the root README.md, we will keep this info as discussed.

File: README.md (hunk 2 above)
Comment (ldclakmal) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (ldclakmal) [reply README.md]: Fixed.

**Pull request merged**